cmd/snap-exec/main_test.go

```diff
@@ -45,20 +46,19 @@ var _ = Suite(&snapExecSuite{})
 
 func (s *snapExecSuite) SetUpTest(c *C) {
 	// clean previous parse runs
-	opts.Command = ""
-	opts.Hook = ""
+	snapExec.SetOptsCommand("")
```

Comment:
Do you need to undo that anywhere? I don't suppose so (just testing) but perhaps a `MockCommandAndHook` thing would suffice?